File: java/chaincue-real-estate/src/main/java/com/chaincuerealestate/chaincuerealestate/services/HouseImageServiceI.java
package com.chaincuerealestate.chaincuerealestate.services;

import com.chaincuerealestate.chaincuerealestate.domains.HouseImage;

import java.util.List;

public interface HouseImageServiceI {
    HouseImage save(String url);

    HouseImage findById(String id);

    List<HouseImage> findAll();
}

File: kotlin/chaincue-real-estate/src/main/kotlin/com/example/chaincuerealestate/services/HouseImageServiceI.kt
package com.example.chaincuerealestate.services

import com.example.chaincuerealestate.domains.HouseImage

interface HouseImageServiceI {
    fun save(url: String): HouseImage
    fun findById(id: String): HouseImage
    fun findAll(): List<HouseImage>
}
